benchmark/profile_generation.py

```diff
@@ -78,14 +78,12 @@ def warmup(model, concurrency: int, input_ids: List[int], warmup_round: int,
         return
 
     print('start to warmup ...')
-    output_seqlen = gen_config.max_new_tokens
```

Comment:
`python benchmark/profile_generation.py /nvme/shared_data/InternLM/internlm2_5-7b-chat --backend pytorch`
--- 
profiling ... concurrency: 1, n_prompt_token: 1, n_completion_token: 128, test_round: 3, warmup_round: 1
Loading checkpoint shards: 100%|████████████████████████████████████████████████████████████████████████████████| 8/8 [00:05<00:00,  1.58it/s]
start to warmup ...
end warmup, elapsed time: 5.39s
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████| 3/3 [00:07<00:00,  2.49s/it]

--------------------------------------------------
total time: 7.46s
concurrency: 1, test_round: 3
input_tokens: 1, output_tokens: 128
first_token latency(min, max, ave): 0.041s, 0.041s, 0.041s
total_token latency(min, max, ave): 2.462s, 2.523s, 2.496s
token_latency percentiles(50%,75%,95%,99%)(s): [0.019, 0.019, 0.037, 0.04]
throughput(output): 51.46 token/s
throughput(total): 51.86 token/s
--------------------------------------------------
Traceback (most recent call last):
  File "benchmark/profile_generation.py", line 487, in <module>
    main()
  File "benchmark/profile_generation.py", line 442, in main
    output = _process_map(profile_target, (args.model_path, ))
  File "benchmark/profile_generation.py", line 385, in _process_map
    raise ret
AttributeError: 'Engine' object has no attribute 'model_name'